src/app/dashboard/components/dashboard-disease-menu/dashboard-disease-menu.component.ts

```diff
@@ -1,8 +1,48 @@
-import { Component } from '@angular/core';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { EntityLink } from '../../../_constants/common';
+import { Disease } from '../../../_constants/enums';
+import { Filter } from '../../../_models/common';
+import { EnumToKeyValuePipe } from '../../../_pipes/enum-to-key-value/enum-to-key-value.pipe';
+import { FilterService } from '../../../_services/filter.service';
 
 @Component({
   selector: 'app-dashboard-disease-menu',
   templateUrl: './dashboard-disease-menu.component.html',
   styleUrls: ['./dashboard-disease-menu.component.scss'],
 })
-export class DashboardDiseaseMenuComponent {}
+export class DashboardDiseaseMenuComponent implements OnInit, OnDestroy {
+  links: EntityLink[] = [];
+  pipe = new EnumToKeyValuePipe();
```

Comment:
don't instantiate a new pipe... if you want to use it in a component class declare it as a provider.
in your case you don't even need that, you should be able to use it directly in your html

Reply:
Resolved.